libs/features/mas/web-components/src/variants/ccd-slice.js

```diff
@@ -0,0 +1,76 @@
+import { VariantLayout } from './variant-layout.js';
+import { html, css } from 'lit';
+import { CSS } from './ccd-slice.css.js';
+
+export class CCDSlice extends VariantLayout {
+
+  getGlobalCSS() {
+    return CSS;
+  }
+
+  renderLayout() {
+    return html` <div class="content">
+          <slot name="icons"></slot> ${this.badge}
+          <slot name="body-s"></slot>
```

Comment:
typo: this should be body-xs

Reply:
or merch-datasource that should be `body-s` too :)

Reply:
I have fixed in merch-datasource, thank you.